gslab_scons/release.py

```diff
@@ -49,6 +50,18 @@ def main(user_yaml = 'config_user.yaml', release_files = []):
     local_release = '%s/%s/' % (release_dir, name)
     local_release = local_release + version + '/'
 
+    # Create provenance file in ./release
+    try:
+        readme = next(arg for arg in sys.argv if re.search("^readme=", arg))
+    except:
+        readme = "readme=./README.md"
+
+    readme = re.sub('^readme=', '', readme)
+    provenance_release_note = '%s, version %s' % (name, version)
+    make_provenance(start_path = '.',
+                    readme_path = readme,
+                    provenance_path = './release/provenance.log',
```

Comment:
I think we should allow a user running release.py to control some of the default options in make_provenance.

Reply:
As of the moment we hard code the release directory to be `./release` so I think this is fine.

Reply:
I meant more about controlling things like `find_for_me` and `detail_limit`.

Reply:
With the exception of `excluded_dirs` (which you have to activate by writing a wrapper for `__main__` rather than calling using `python -m` the rest can now be specified with command line args.

Reply:
Make sure we mention the command line args somewhere.